code/src/index.css

```diff
@@ -5,9 +5,169 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(24, 24, 20);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+h1 {
+  color:white;
+  font-size: 1.5em;
+  margin: 30px 50px;
+}
+h2 {
```

Comment:
We would suggest to try to be consistent with the spaces between the different css parts. It makes it easier to read if it is a space between them all! Whatever you chose to do, try to be consistent =)